spec/features/01_can_create_account_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature 'can create a new user account' do
```

Comment:
This line might be more explicit. It almost looks like a scenario. I might do something more like "User can create new account'. That's a bit nit picky/ opinion so feel free to chat if you have a reason you want it this way!